sample_metadata/parser/generic_metadata_parser.py

```diff
@@ -555,6 +566,7 @@ async def get_sequence_meta(
         read_filenames: List[str] = []
         gvcf_filenames: List[str] = []
         read_checksums: List[str] = []
+        reference_fastas: set[str] = set()
```

Comment:
Should this be called reference assembly as well, for consistency? (Or the other way around.)

Reply:
Yes, absolutely :)